src/core/reducers/addons.js

```diff
@@ -366,6 +465,36 @@ export default function addonsReducer(
       }
       return state;
     }
+
+    case FETCH_ADDON_INFO: {
+      const { slug } = action.payload;
+      return {
+        ...state,
+        infoBySlug: {
+          ...state.infoBySlug,
+          [slug]: {
+            info: null,
```

Comment:
Typically we reserve `null` for when the data *was* fetched but it does not exist. You can leave this as is for now, though, because add-on info can always be fetched. I just wanted to mention it. For cases like this, `undefined` would be more consistent with other reducers.

Reply:
Well, actually, you might need `null` for the case of a 404. Anyway, we could adjust it later as needed.